.eslintrc.json

```diff
@@ -0,0 +1,13 @@
+{
+  "extends": "airbnb-base",
+  "rules": {
+    "max-len": ["error", {"code": 200, "ignoreComments": true}],
+    "no-unused-vars": "off",
```

Comment:
Were there specific places that having unused variables was desired? I feel like this may be a bad rule to disable.

Reply:
I just checked. Seems it was yelling at him whenever the automatically generated includes/requires for each file were not always used for every model.

The rule definitely should be there in that case. Otherwise we'd have to somehow tailor each's includes/requires around the specific model.

Reply:
Exactly right, we have imports in each model that it was complaining about. I figured it was reasonable to disable it for just this repo though, since it's only dealing with models, but I'm open to debate about it.

Reply:
There are ways to tell eslint to ignore variable names that match a regex. I wonder if it'd be easy enough to write one that matches those imports, and only those?

Reply:
If not, I'm fine with leaving it disabled.

Reply:
Yeah, that is probably the way to go. I've just pushed something that re-enabled this check, but ignores those declarations.